x-pack/plugins/reporting/public/components/reporting_panel_content.tsx

```diff
@@ -157,34 +181,71 @@ export class ReportingPanelContent extends Component<Props, State> {
   };
 
   private createReportingJob = () => {
+    const { intl } = this.props;
+
     return reportingClient
       .createReportingJob(this.props.reportType, this.props.getJobParams())
       .then(() => {
         toastNotifications.addSuccess({
-          title: `Queued report for ${this.props.objectType}`,
-          text: 'Track its progress in Management',
+          title: intl.formatMessage(
+            {
+              id: 'xpack.reporting.panelContent.notification.successTitle',
+              defaultMessage: 'Queued report for {objectType}',
+            },
+            { objectType: this.props.objectType }
+          ),
+          text: (
+            <FormattedMessage
+              id="xpack.reporting.panelContent.notification.successDescription"
+              defaultMessage="Track its progress in Management"
+            />
+          ),
           'data-test-subj': 'queueReportSuccess',
         });
         this.props.onClose();
       })
       .catch((kfetchError: KFetchError) => {
         if (kfetchError.message === 'not exportable') {
           return toastNotifications.addWarning({
-            title: `Only saved ${this.props.objectType} can be exported`,
-            text: 'Please save your work first',
+            title: intl.formatMessage(
+              {
+                id: 'xpack.reporting.panelContent.notification.notExportableTitle',
+                defaultMessage: 'Only saved {objectType} can be exported',
+              },
+              { objectType: this.props.objectType }
+            ),
+            text: (
+              <FormattedMessage
+                id="xpack.reporting.panelContent.notification.notExportableDescription"
+                defaultMessage="Please save your work first"
+              />
+            ),
           });
         }
 
         const defaultMessage =
-          kfetchError.res.status === 403
-            ? `You don't have permission to generate this report.`
-            : `Can't reach the server. Please try again.`;
+          kfetchError.res.status === 403 ? (
+            <FormattedMessage
+              id="xpack.reporting.panelContent.notification.noPermissionDescription"
```

Comment:
Maybe do not use .notification namespace?
Also noPermissionDescription => noPermissionToGenerateReportDescription